kentikapi/internal/api_resources/devices_test.go

```diff
@@ -112,7 +113,7 @@ func TestCreateDeviceRouter(t *testing.T) {
 	models.SetOptional(&router.DeviceBGPPassword, "bgp-optional-password")
 	models.SetOptional(&router.SiteID, 8483)
 	models.SetOptional(&router.DeviceBGPFlowSpec, true)
-	device, err := devicesAPI.Create(nil, *router)
```

Comment:
StubTransport doesnt use the context

Reply:
Ok, but I did this change to suppress the linter output. And I guess this check is based on the Context package documentation (https://pkg.go.dev/context@go1.16#pkg-overview):
> Do not pass a nil Context, even if a function permits it. Pass context.TODO if you are unsure about which Context to use. 

Do you suggest to disable this check globally / locally?
In my opinion the best option here is just to pass `Background()` here and move on.

Reply:
agree, no point in fighting the tool here